packages/mediacenter/kodi/package.mk

```diff
@@ -32,7 +32,7 @@ PKG_LONGDESC="Kodi Media Center (which was formerly named Xbox Media Center or X
 PKG_IS_ADDON="no"
 PKG_AUTORECONF="no"
 
-PKG_CMAKE_SCRIPT="$ROOT/$PKG_BUILD/project/cmake/CMakeLists.txt"
+PKG_CMAKE_SCRIPT="$ROOT/$PKG_BUILD/CMakeLists.txt"
```

Comment:
PKG_CMAKE_SCRIPT is not needed now. $ROOT/$PKG_BUILD/CMakeLists.txt is the default